test/common/upstream/cluster_manager_impl_test.orig.cc

```diff
@@ -0,0 +1,6614 @@
+#include "envoy/admin/v3/config_dump.pb.h"
```

Comment:
Oh, I guess this PR is still WIP.
Feel free to ping me once this is ready for review